core/hw/naomi/naomi_cart.cpp

```diff
@@ -84,7 +84,7 @@ bool naomi_cart_LoadRom(char* file)
 	}
 	fclose(fl);
 
-	printf("+%d romfiles, %.2f MB set size, %.2f MB set address space\n", files.size(), setsize / 1024.f / 1024.f, RomSize / 1024.f / 1024.f);
+	printf("+%lu romfiles, %.2f MB set size, %.2f MB set address space\n", files.size(), setsize / 1024.f / 1024.f, RomSize / 1024.f / 1024.f);
```

Comment:
is `%lu` the right option for all platforms? windows long is 32 bits long, even in 64-buils (LLP model).